developers/weaviate/benchmarks/ann.md

```diff
@@ -233,7 +183,74 @@ import AnnGist960 from '/_includes/ann-gist-960.mdx';
 
 <AnnReadResultsTable/>
 
-## Learn more & FAQ
+
+#### Recommended configuration for GIST 960 
+Below, you can see the recommended configuration for this dataset, which is an opinionated pick about a good recall/latency/throughput trade-off to give you a good overview of Weaviate's
+performance with this dataset. 
+
+| `efConstruction` | `maxConnections` | `ef` | **Recall@10** | **QPS (Limit 10)** | **Mean Latency (Limit 10**) | **p99 Latency (Limit 10)** |
+| ----- | ----- | ----- | ----- | ----- | ----- | ----- | 
+| 512 | 32 | 128 | 94.14% | 1935 | 15.05ms | 19.86ms |
+
+</TabItem>
+</Tabs>
+
+
+
+
+
+## Benchmark Setup
+
+### Scripts
+
+This benchmark is produced using [open source
+scripts](https://github.com/weaviate/weaviate-benchmarking), so you can reproduce it yourself.
+
+### Hardware
+
+![Setup with Weaviate and benchmark machine](/img/docs/weaviate_benchmark_setup.png)
+
+For the purpose of this benchmark we've used two GCP instances within the same VPC:
+
+* **Benchmark** – a `c2-standard-30` instance with 30 vCPU cores and 120 GB memory – to host Weaviate.
+* **Script** – a smaller instance with 8 vCPU – to run benchmarking scripts.
+
+💡 the `c2-standard-30` was chosen for benchmarking for two reasons:
+
+* It is large enough to show that Weaviate is a highly-concurrent vector search
+  engine and scales well while running thousands of searches across multiple threads.
+* It is small enough to represent a typical production case without inducing
+  high costs.
+
+Based on your throughput requirements, it is very likely that you will run Weaviate
+on a considerably smaller or larger machine in production.
+
+In [this section below](#what-happens-if-i-run-with-fewer-or-more-cpu-cores-than-on-the-example-test-machine)
+we have outlined what you should expect when altering the configuration or
+setup parameters.
+
+### Experiment Setup
+
+The selection of datasets is modeled after
+[ann-benchmarks](https://github.com/erikbern/ann-benchmarks). The same test
+queries are used to test speed, throughput, and recall. The provided ground
+truths are used to calculate the recall.
+
+The imports were performed using Weaviate's Python clients. The concurrent
+(multi-threaded) queries were measured using Go. Each language may have a
+slightly different performance, and you may experience different results if you
+send your queries using another language. For the maximum throughput, we
+recommend using the [Go](/developers/weaviate/client-libraries/go.md) or
+[Java](/developers/weaviate/client-libraries/java.md) clients.
+
+The complete import and test scripts are available [here](https://github.com/weaviate/weaviate-benchmarking).
+
+## Learn more & Benchmark FAQ
+
+### How can I get the most performance for my use case?
+If your use case is similar to one of the showcased datasets this benchmark, you can start with the recommended configurations for the HNSW parameters.
```

Comment:
Consider:
If your use case is similar to one of the benchmark tests, use the recommended HNSW parameter configurations to start tuning.